content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+# Sourcegraph 4.0
+
+> NOTE: This post is very WIP. It's a rough draft of what individual teams intend to ship.
+
+- [From code search toward a code intelligence platform](/blog/code-search-to-code-intelligence) - the longer-term vision/story
+- [Secure, easy, and scalable Sourcegraph Cloud managed instances for the enterprise](/blog/enterprise-cloud) - the Cloud announcement (including SOC2 Type 2, etc.) that coincides with 4.0
+
+## TL;DR
+
+`TODO(sqs): The section headers with emojis are what I think we should use, but the wording of the bullet points within are WIP`
+
+##### 🌅 Code intelligence: more useful, more times daily for every dev
+
+### A faster, simpler search experience 
+We’ve improved the search UI to make the platform more intuitive with small usability improvements while making it easier for new users to get started with tips and recommendations on the revamped homepage. Performance across the platform saw a boost by using the latest in React, lazy loading, and priority rendering. These usability and performance enhancements help you get quick answers about your codebase and get unblocked faster. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Search]
+
+Docs: provide a link to the updated docs
+
+- Smarter code search
+### Understand usage and search structure with high-level aggregations of search results
+High-level aggregations of search results powered by Code Insights are now displayed in addition to results. Responses to the query are now grouped by location (repo or file), author, and arbitrary capture group pattern to help answer to questions such as:
+
+- Which files have the bulk of these search results?
+- Which repositories is this library used in most?
+- Which subcomponents are used most from this library?
+- Who is using our new innersource library?
+- Which areas of the code were added back in these regressions?
+- What are all the possible arguments we can pass to this function?
+- Aggregation can easily be reused by saving the visualization to your Code Insights dashboard.
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Insights]
+
+Docs: provide a link to the updated docs
```

Comment:
@Joelkw please provide media and a link to updated docs when ready

Reply:
Docs are at: https://docs.sourcegraph.com/code_insights/explanations/search_results_aggregations

I'm also working on an 'example use cases' doc I'd like to link, but that won't be ready until after cancun. 

And here is a video (since we aren't using gifs) we can use for the image: https://www.loom.com/share/83e156c214da415bb1f8726afea5fe00

Reply:
(edit, updated video link to correct one) https://www.loom.com/share/0e6c5f893b374fe8b960ee4cd353f746